src/client/firefox/create.js

```diff
@@ -21,7 +21,10 @@ export function createFrame(frame: FramePacket): ?Frame {
   if (frame.type == "call") {
     const c = frame.callee;
     title =
-      c.name || c.userDisplayName || c.displayName || L10N.getStr("anonymous");
+      c.name ||
+      c.userDisplayName ||
+      c.displayName ||
+      L10N.getStr("anonymousFunction");
```

Comment:
do we need `L10N.getStr("anonymousFunction");` if we're going to add it later when we format the display name?

Reply:
probably not, let's remove it.